### ISSUE TITLE
Material Design Icons v7 (mdi-v7) icon sets are missing

### ISSUE THREAD

**Issue by susnux** [quasarframework/quasar#14696]
### What happened?

Using `mdi-v7` leads to this error:
```
App •  WAIT  • Compiling of "UI" in progress...
 App •  DONE  • "UI" compiled with errors • 13839ms

 App •  ERROR  •  UI  in ./.quasar/quasar-user-options.js

Module not found: Can't resolve imported dependency "quasar/icon-set/mdi-v7"
Did you forget to install it? You can run: npm install --save quasar/icon-set/mdi-v7


 App •  COMPILATION FAILED  • Please check the log above for details.

quasar build  32,17s user 0,97s system 216% cpu 15,276 total
1 
```

### What did you expect to happen?

No error, `mdi-v7` should be useable.

### Reproduction URL

https://stackblitz.com/edit/quasarframework-webpack-yvqbbx?file=quasar.config.js

### How to reproduce?

1. Update `@quasar/extras` to 1.15.5
2. Configure to use `mdi-v7`
3. Try to build project

### Flavour

Quasar CLI with Webpack (@quasar/cli | @quasar/app-webpack)

### Areas

Quasar Extras (@quasar/extras)

**Comment by MilosPaunovic** [comment]
Using `mdi-v7` is not yet supported through the `@quasar/extras`.

**Comment by susnux** [comment]
Well then the documentation is wrong as it states that `mdi-v7` is supported with extras 1.15+

https://github.com/quasarframework/quasar/blame/dev/extras/README.md#L33

**Comment by MilosPaunovic** [comment]
Ohh, I do apologize, I must've missed something. Let's wait for others to weigh in.

**Comment by yusufkandemir** [comment]
It's correct that MDI v7 itself is supported on extras v1.15+, but the `quasar` package doesn't have any icon sets for it. So, you can add it to `quasar.config.js > extras`, import SVG icons, etc. but can't do `quasar.config.js > framework > iconSet = 'mdi-v7' // nor 'svg-mdi-v7'` yet. According to README of extras, they are supposed to exist. So, I would consider this a bug. I think the problem is now clear.

**Comment by helioh3** [comment]
I have this is problem too. I use the version v1. Any update Quasar-extra? Thanks!

**Comment by rstoenescu** [comment]
The Quasar IconSet mdi-v7 will be available in v2.10.2 and v1.22.2
